src/controller/DonationsController.ts

```diff
@@ -20,28 +22,64 @@ export class DonationsController {
       return `Webhook Error: ${err.message}`;
     }
 
-    const { type, data } = request.body;
+    // const { type, data } = request.body;
```

Comment:
Can just delete this right?

Reply:
Yep--I think that came in from a merge, but it was unused--I'll remove!